include/transaction.inc.php

```diff
@@ -612,10 +612,11 @@ public function get_transactions($height = "", $id = "")
         if (empty($id) && empty($height)) {
             return false;
         }
+        $version_limit = $includeMiningRewards ? 0 : 1;
         if (!empty($id)) {
-            $r = $db->run("SELECT * FROM transactions WHERE block=:id AND version>0", [":id" => $id]);
+            $r = $db->run("SELECT * FROM transactions WHERE block=:id AND version >= :version", [":id" => $id, ":version" => $version_limit]);
```

Comment:
```suggestion
            $r = $db->run("SELECT * FROM transactions WHERE block=:id AND version >= :version", [":id" => $id, ":version" => $versionLimit]);
```

I was also wondering if this section should be simplified as the following? No reason to though.
```php
$r = !empty($id) ?
    $db->run(
        "SELECT * FROM transactions WHERE block=:id AND version >= :version",
        [":id" => $id, ":version" => $versionLimit]
    ) :
    $db->run(
        "SELECT * FROM transactions WHERE height=:height AND version >= :version",
        [":height" => $height, ":version" => $versionLimit]
    );
```